src/components/Trackpad/ControlBar.tsx

```
@@ -60,30 +60,42 @@ export const ControlBar: React.FC<ControlBarProps> = ({
 			>
 				{scrollMode ? "Scroll" : "Cursor"}
 			</button>
+			<button
+				className="btn btn-sm btn-outline"
+			>
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)